.github/workflows/build_and_test.yml

```diff
@@ -14,19 +14,44 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build_and_test_with_code_coverage_module:
-    name: Build, test and upload code coverage by module
-    runs-on: ubuntu-latest
+  build_save-cloud-common:
```

Comment:
could it be more compact via some matrix?

where matrix is `[save-cloud-common, save-orchestrator-common, ...]`

Reply:
I need to check input.<module_name> to run building of a specific module.
I'm not sure GitHub allows to dynamic variable to get some value from `inputs`, but I can construct a martix dynamically.
Will try to refactor

But still -- backend and sandbox will be separated because I need to specify a dependency to save-cli artifact

Reply:
done

Reply:
oh..missed that you asked about `build_and_test` -- common modules should be built one by one (each next module takes cache from previous one and updates it). It needs to allow to use caches in rest steps.

I added a similar approach with matrix (like for left modules) in deploy_images.yml: There backend and sandbox are built with dependency to save-cli and rest are built independently